apps/rush-lib/assets/rush-init/.gitignore

```diff
@@ -0,0 +1,68 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# Runtime data
+pids
+*.pid
+*.seed
+*.pid.lock
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# nyc test coverage
+.nyc_output
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
+
+# Bower dependency directory (https://bower.io/)
+bower_components
+
+# node-waf configuration
+.lock-wscript
+
+# Compiled binary addons (https://nodejs.org/api/addons.html)
+build/Release
+
+# Dependency directories
+node_modules/
+jspm_packages/
+
+# Optional npm cache directory
+.npm
+
+# Optional eslint cache
+.eslintcache
+
+# Optional REPL history
+.node_repl_history
+
+# Output of 'npm pack'
+*.tgz
+
+# Yarn Integrity file
+.yarn-integrity
+
+# dotenv environment variables file
+.env
+
+# next.js build output
+.next
+
+# Common toolchain intermediate files
+lib
+lib-amd
+lib-es6
+temp
```

Comment:
dist?

Reply:
Being a general template, I feel that the initial rules should be very conservative.  People can easily add more exclusions later, but it's a bad experience to discover that a file has mysteriously not been added to Git.

So rather than adding "dist" I'm going to remove lib*.  People who don't use a compiler often author files directly in these folders.

---
In reply to: [217269179](https://github.com/Microsoft/web-build-tools/pull/823#discussion_r217269179) [](ancestors = 217269179)